meta-refkit/conf/distro/include/refkit-supported-recipes.txt

```diff
@@ -84,6 +85,7 @@ freetype@core
 gawk@core
 gcc-runtime@core
 gcc@core
+gconf@core
```

Comment:
it is required by some module that is included in the oe-recipe. I think it can be disabled from configure. Can try that tomorrow...